### PR TITLE
port lidar_centerpoint

### PR DESCRIPTION
**Pull request by yukke42**
## Description

.auto porting
- autoware_perception_msgs -> autoware_auto_perception_msgs.

I checked that detected objects are published.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]

File: perception/object_recognition/detection/lidar_centerpoint/src/node.cpp [hunk 1]
```diff
@@ -90,46 +89,48 @@ void LidarCenterPointNode::pointCloudCallback(
     float h = boxes3d_vec[obj_i * Config::num_box_features + 7];
     float yaw = boxes3d_vec[obj_i * Config::num_box_features + 8];
 
-    autoware_perception_msgs::msg::DynamicObjectWithFeature feature_obj;
-    autoware_perception_msgs::msg::Semantic semantic;
+    autoware_auto_perception_msgs::msg::DetectedObject obj;
+    autoware_auto_perception_msgs::msg::ObjectClassification classification;
+    obj.existence_probability = score;
```
Comment (yukkysaito) [inline perception/object_recognition/detection/lidar_centerpoint/src/node.cpp:94]: コメントで
// TODO 確率ではないが一時的な処置であり、scoreはclassificationのsoftmaxの結果
みたいな記述を入れたほうがいいかもす

**Review by yukkysaito — CHANGES_REQUESTED** [review]

File: perception/object_recognition/detection/lidar_centerpoint/src/node.cpp (hunk 1 above)
Comment (yukkysaito) [reply perception/object_recognition/detection/lidar_centerpoint/src/node.cpp:94]: 一応TODOであくまでもDNNで推定しているのはconfidenceであり、確率ではないという文言を入れてもらえますか？
readmeにもあるといいかもです

Comment (yukke42) [reply perception/object_recognition/detection/lidar_centerpoint/src/node.cpp:94]: fixed at [a65da99](https://github.com/tier4/AutowareArchitectureProposal.iv/pull/527/commits/a65da99b0df6b5568dc69be5b09b2b9315811ec3)

Comment (yukke42) [reply perception/object_recognition/detection/lidar_centerpoint/src/node.cpp:94]: fixed at [61be94e](https://github.com/tier4/AutowareArchitectureProposal.iv/pull/527/commits/61be94eb61f4f3344ce3927a7b7a6d14311b464f)

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**